pkgs/applications/audio/mopidy/subidy.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, pythonPackages, mopidy }:
+
+pythonPackages.buildPythonApplication rec {
+  pname = "mopidy-subidy";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "Prior99";
+    repo = pname;
+    rev = version;
+    sha256 = "0c5ghhhrj5v3yp4zmll9ari6r5c6ha8c1izwqshvadn40b02q7xz";
+  };
+
+  propagatedBuildInputs = [ mopidy pythonPackages.py-sonic];
+
+  checkInputs = with pythonPackages; [ pytestCheckHook ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://www.mopidy.com/";
+    description = "Mopidy extension for playing music from a Subsonic-compatible Music Server";
+    license = licenses.bsd3;
+    maintainers = with maintainers; [ wenngle ];
+    hydraPlatforms = [];
```

Comment:
Why shouldn't it be build on the hydra?

Reply:
That was a mistake; I initially used one of the other Mopidy extensions as a template for this one.